model/post.go

```diff
@@ -101,6 +101,7 @@ type Post struct {
 	ReplyCount   int64         `json:"reply_count" db:"-"`
 	LastReplyAt  int64         `json:"last_reply_at" db:"-"`
 	Participants []*User       `json:"participants" db:"-"`
+	IsFollowing  bool          `json:"is_following" db:"-"` // for root posts in collapsed thread mode indicates if the currect user is following this thread
```

Comment:
current or correct?

Reply:
@agnivade I said "currect" and I stand by it! :)
I'll fix the typo.